deployments/ecmps/release-v2.0-fix/Sprint8/6111/camdecmps/functions/rpt_quartely_em_summary_data.sql

```diff
@@ -0,0 +1,331 @@
+-- FUNCTION: camdecmps.rpt_quartely_em_summary_data(text, numeric)
+
+DROP FUNCTION IF EXISTS camdecmps.rpt_quartely_em_summary_data(text, numeric) CASCADE;
+
+CREATE OR REPLACE FUNCTION camdecmps.rpt_quartely_em_summary_data(
+	monplanid text,
+	vyear numeric)
+    RETURNS TABLE(location character varying, "calendar_year" numeric, "period" text, "op_hours" numeric, "op_days" numeric, "op_time" numeric, "hit" numeric, "so2m" numeric, "co2m" numeric, "noxr" numeric, "noxm" numeric, "bco2" numeric) 
+    LANGUAGE 'plpgsql'
+
+    COST 100
+    VOLATILE 
+    ROWS 1000
+    
+AS $BODY$
+DECLARE 
+    monLocIds character varying[];
+
+BEGIN
+    RETURN QUERY
+    (with lst as
+        (
+            select  pln.mon_plan_id,
+                    prd.rpt_period_id,
+                    mpl.mon_loc_id,
+                    pln.fac_id,
+                    prd.calendar_year,
+                    prd.quarter
+            from  camdecmps.monitor_plan pln,
+                    camdecmps.monitor_plan_location mpl,
+                    camdecmpsmd.reporting_period prd
+            where  pln.mon_plan_id = monplanid
+            and  mpl.mon_plan_id  = pln.mon_plan_id
+            and  (vyear is null or prd.calendar_year = vyear)
+        )
+        select
+                coalesce( unt.unitid, stp.stack_name) as location,
+                cmb.calendar_year,
+                cmb.Period,
+                cmb.Op_Hours,
+                cmb.Op_Days,
+                cmb.Op_Time,
+                cmb.Hit,
+                cmb.So2m,
+                cmb.Co2m,
+                cmb.Noxr,
+                cmb.Noxm,
+                cmb.Bco2
+        from  (
+                    --
+                    -- Quarterly Data for the Non-Fuel-Specific and Fuel-Specific Report Report Tables
+                    --
+                    select  sel.mon_plan_id,
+                            sel.fuel_ind,
```

Comment:
FUEL_IND does not appear to be used and is not needed since it should always be 0 for the non-fuel-specific information.